lib/config/folderconfiguration.go

```diff
@@ -253,3 +263,7 @@ func (f *FolderConfiguration) CheckAvailableSpace(req int64) error {
 	}
 	return fmt.Errorf("insufficient space in %v %v", fs.Type(), fs.URI())
 }
+
+func (f *FolderConfiguration) HasPlaintextData() bool {
```

Comment:
call `IsPlainText`? Because the current signature may imply that the folder may be half-encrypted.

Also, sometimes `FolderTypeReceiveEncrypted`, sometimes `HasPlaintextData()` is used to detect the folder type. Perhaps we don't need this function at all?